helpers/helper.js

```diff
@@ -0,0 +1,16 @@
+import { formatDistance, format } from 'date-fns';
+const helper = {
+  name: 'helper',
+  getDuration(from, to) {
+    const date1 = new Date(from);
+    const date2 = new Date(to);
+
+    return formatDistance(date2, date1, { addSuffix: true });
+  },
+  formatDate(val) {
+    return format(new Date(val), 'dd.MM.yyyy');
+  },
+};
+export default ({ app }, inject) => {
+  inject('helper', helper);
```

Comment:
No need to inject it as we can use it just like a utility helper method. But it's optional.